packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -272,6 +276,9 @@ const makeRenderer = (
 					media: getMedia(userAgent, userAgentDevice),
 					browserId: parseBrowserIdCookie(state),
 					clientIp,
+					siftSessionId: parseSiftSessionCookie(state), // userSession cookie created in fastly and used by sift science to correlate spammy
+					// behavior to user activity; cookie has an expiration time of 4 hours and should be
+					// reset on logout
```

Comment:
Adding a link to documentation or at least to Sift would be helpful.

Reply:
+1 - also let's try to keep 'sift' cookie documentation in one place if possible - add comments to function in `cookieUtils`, but don't specify cookie expiration, because that's 'owned'/specified elsewhere.

Reply:
Make sense. Updated in recent commit.